appserver/tests/quicklook/ejb/singleton/build.properties

```diff
@@ -43,7 +43,7 @@ testsuite.name=EJB_Singleton
 testng.test.name=EJB_Singleton
 #possible choices war|ear|jar
 contextroot=singleton
-archive.type=ear
+archive.type=war
```

Comment:
Somewhat unrelated, EJB_Singleton was failing due to the file not being generated, changing to war generates the file and passes the test. Happy to revert if neccessery.